go/src/koding/kites/config/konfig.go

```diff
@@ -170,6 +170,7 @@ type Konfig struct {
 	PublicBucketRegion string `json:"publicBucketRegion,omitempty"`
 
 	LockTimeout time.Duration `json:"lockTimeout,omitempty"`
+	NoMetrics   bool          `json:"noMetrics,string,omitempty"`
```

Comment:
why just not:

```
$ kd config set metrics false
```

Generally, I always have problems when reading double negation (operator-name) like
`if !noSomething {`. 🤔